contao/config/event_listeners.php

```diff
@@ -0,0 +1,37 @@
+<?php
+/**
+ * The MetaModels extension allows the creation of multiple collections of custom items,
+ * each with its own unique set of selectable attributes, with attribute extendability.
+ * The Front-End modules allow you to build powerful listing and filtering of the
+ * data in each collection.
+ *
+ * PHP version 5
+ *
+ * @package    MetaModels
+ * @subpackage AttributeText
+ * @author     Christopher Boelter <christopher@boelter.eu>
+ * @copyright  The MetaModels team.
+ * @license    LGPL.
+ * @filesource
+ */
+
+use MetaModels\Attribute\Events\CreateAttributeFactoryEvent;
+use MetaModels\Attribute\Text\AttributeTypeFactory;
+use MetaModels\MetaModelsEvents;
+use MetaModels\Events\MetaModelsBootEvent;
+use MetaModels\DcGeneral\Events\Table\InputScreens\BackendSubscriber;
```

Comment:
puuh the same thing with every change, too much attributes :)
But I think it will be very useful to restructrure the namespaces for the future. At the moment I will move my class to the attribute namespace and later we change it. Ok?